spec/requests/api/v1/content_spec.rb

```diff
@@ -240,6 +240,48 @@
     end
   end
 
+  describe 'Filter by all organisations' do
+    let(:another_org_id) { 'c97bbc95-967a-4678-848b-6f393171f194' }
+    let(:edition1) { create :edition, organisation_id: organisation_id, date: 15.days.ago }
+    let(:edition2) { create :edition, organisation_id: another_org_id, date: 15.days.ago }
+
+    subject { get '/content', params: { date_range: 'past-30-days', organisation_id: 'all' } }
+
+    before do
```

Comment:
Do we need a `before` block for a single example?

Reply:
I don't see this as a problem.